Update 15 (#277)
NilPointer-Software/bdfd-wiki: src/bdscript/channelName.md

# $channelName

> Returns a name of the provided channel ID.

Reviewer: ```suggestion
Returns the name for the provided channel ID.
```

## Syntax
```
$channelName[channel ID]
```
### Parameters

> - `channel ID` `(Type: Snowflake || Flag: Required)`: The id of the channel whose name is to be returned.

Reviewer: ```suggestion
- `channel ID` `(Type: Snowflake || Flag: Required)`: The ID of the channel whose name will be returned.
```
